<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1026</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="9">
     <widget class="QLabel" name="info">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Program wylicza ile czasu spędzasz przed ekranem:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QLabel" name="info1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Podaj ile godzin spędzasz przed ekranem:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QSpinBox" name="input_godzin_spinBox">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="1" column="10" rowspan="2" colspan="7">
     <widget class="QLabel" name="label_invisible">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>40</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>40</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>40</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>255</red>
            <green>40</green>
            <blue>2</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="PlaceholderText">
          <brush brushstyle="SolidPattern">
           <color alpha="128">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTipDuration">
       <number>-4</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="info1_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ustawienie limitu:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="6">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>godzin bezpiecznego czasu przed komputerem</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="7">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wyliczona ilość procentowa z doby wynosi:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QLabel" name="outputWiget_procent_doba">
      <property name="text">
       <string>0,00</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Ilość przysługujących przerw:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QLabel" name="output_przerwa">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8" colspan="3">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Ilość przerw:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="11">
     <widget class="QSpinBox" name="input_ilosc_przerw_spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="13" colspan="3">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Ilość godzin do przerwy:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="16">
     <widget class="QSpinBox" name="input_ilosc_do_przerwy_spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Łączny czas przerw w minutach:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="2">
     <widget class="QLabel" name="output_czas_przerw">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="8" colspan="3">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Długość przerwy w minutach:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="11">
     <widget class="QSpinBox" name="input_dlugosc_przerwy_spinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="9">
     <widget class="QLabel" name="info_2">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sprawdźmy ile tygodniowo spedzasz przed komputerem:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Poniedziałek:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="input_pon_spinBox">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6" colspan="3">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>W sumie w tygodniu spędzasz:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Wtorek:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="input_wto_spinBox_2">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6" colspan="7">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Średnia ilość godzin w tygodniu przed komputerem  wynosi:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="13">
     <widget class="QLabel" name="output_srednia">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="14">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Środa</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="input_sro_spinBox_3">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="9" column="3" colspan="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Czwartek:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="input_czw_spinBox_4">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="10" column="3" colspan="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="10" column="15" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Przelicz</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Piątek:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="input_pia_spinBox_5">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="11" column="3" colspan="2">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="11" column="15" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Sobota:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="input_sob_spinBox_6">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="12" column="3" colspan="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Niedziela:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="input_nie_spinBox_7">
      <property name="maximum">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="13" column="3" colspan="2">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="input_limit_spinBox">
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QLabel" name="output_tydzien_godzin">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="10">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>godzin</string>
      </property>
     </widget>
    </item>
    <item row="12" column="15" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Wykres</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Czas Start</string>
      </property>
     </widget>
    </item>
    <item row="11" column="7" colspan="2">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Czas Stop</string>
      </property>
     </widget>
    </item>
    <item row="10" column="9" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_czas_start">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="9" colspan="2">
     <widget class="QLCDNumber" name="lcdNumber_czas_stop">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="8">
     <widget class="QLabel" name="output_czas_sekundy">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Zmierzony czas:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="9">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>w sekundach</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
